### DISCUSSION TITLE
Any way to change default code block language?

### DISCUSSION THREAD

**Discussion by jat255** [TriliumNext/Trilium#4207] [Q&A]
I am using this nice syntax highlighting widget for live highlighting of code blocks: https://github.com/antoniotejada/Trilium-SyntaxHighlightWidget.

Due to its design, it does not highlight on code blocks set to "Plain Text", which seems to be the default when creating a new code block by typing three back-ticks (```).

I'm wondering, is there any way to change the language that is used when those three back ticks are entered? It would save a bit of an annoyance having to manually change the language every time I create a new code block.

**Comment by zerebos** [comment]
There's not really a way to do this, no. You'd be better off making a change to that widget where you set the default language for the whole note using an attribute.
